include/Metabox/Posts/CategoryToTag.php

```diff
@@ -30,19 +30,40 @@ public function render() {
             <table class="optiontable">
                 <tr>
                     <td scope="row">
-						<?php
-						wp_dropdown_categories( array(
-							'name'         => 'smbm_mct_cat',
-							'show_count'   => true,
-							'hierarchical' => true,
-							'orderby'      => 'NAME',
-							'hide_empty'   => false,
-						) );
-						?>
+                    	<?php
```

Comment:
Move this to the helper function.

Reply:
You are now missing some of the parameters that we had earlier to the `wp_dropdown_categories` function.

Reply:
@rajanit2000 This comment is not addressed.